### ISSUE TITLE
Support for moving functions?

### ISSUE THREAD

**Issue by donald-s** [jurassix/atom-javascript-refactor#140]
Does this module support moving functions from one file to another? If not, would there be any interest in a PR for this functionality?

**Comment by jurassix** [comment]
Absolutely. This functionality is definitely possible. Forking https://github.com/jurassix/refactoring-codemods to add a new transform is a good starting point.

Once the codemod is working then we need something added to the Atom ui that can trigger this on move.

**Comment by donald-s** [comment]
Fantastic. I'll check it out and work on a PR. How would you like to track this? Shall we create an issue?

**Comment by jurassix** [comment]
Yeah issue and/or open a PR and keep committing along the way
